ee/maintained-apps/outputs/zoom/windows.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
@iansltx @allenhouchins my change to this output to main was just adding the `upgrade_code` just like the other 2 output jsons. This diff has me adding the file back completely. Did we delete this file in the rc and not in `main` for an important reason?

Reply:
Zoom for Windows got added as an FMA between RC cut and now. Adding the file wholesale doesn't hurt anything, so we're good to go here.

Reply:
Follow up here: https://github.com/fleetdm/fleet/pull/36389